.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
-    "restructuredtext.confPath" : "${workspaceRoot}/doc/manual/de",
+    "restructuredtext.confPath" : "${workspaceFolder}/doc/manual/de",
```

Comment:
Is this a required change to fuix the setup or something the visual studio code has changed by itself?

Reply:
changed by vs code